### PR TITLE
[ROT] Rise of Tiamat

### PR DESCRIPTION
**Pull request by kgirtxd**

### CODE REVIEW

File: bestiary/rise-of-tiamat.xml [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+		<name>Rise of Tiamat Bestiary</name>
+		<update version="0.0.1">
+			<file name="rise-of-tiamat.xml" url="https://raw.githubusercontent.com/Ouroboros-dnd/bestiary/master/bestiary/rise-of-tiamat.xml" />
+		</update>
+	</info>
+
+	<element name="Dragonclaw" type="Bestiary" source="Rise of Tiamat" id="ID_WOTC_ROT_STATBLOCK_DRAGONCLAW">
+		<description>
+			<h3 style="margin-bottom:-20px;margin-top:-5px">Dragonclaw</h3>
+			<p style="margin-bottom:-5px"><em>Medium humanoid (human), neutral evil</em></p>
```
Comment (ziphmose) [inline bestiary/rise-of-tiamat.xml:13]: Lawful Evil

Comment (kgirtxd) [reply bestiary/rise-of-tiamat.xml:13]: image
?

File: bestiary/rise-of-tiamat.xml [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+				<li><strong>Armor Class</strong> 15 (studded leather)</li>
+				<li><strong>Hit Points</strong> 78 (12d8 + 24)</li>
+				<li><strong>Speed</strong> 30 ft.</li>
+			</ul>
+			<table style="text-align:center;margin-top:10px;margin-bottom:10px">
+				<thead><tr><td>STR</td><td>DEX</td><td>CON</td><td>INT</td><td>WIS</td><td>CHA</td></tr></thead>
+				<tr><td>11 (+0)</td><td>16 (+3)</td><td>14 (+2)</td><td>12 (+1)</td><td>12 (+1)</td><td>14 (+2)</td></tr>
+			</table>
+			<ul class="unstyled">
+				<li><strong>Saving Throws</strong> Wis +3</li>
```
Comment (ziphmose) [inline bestiary/rise-of-tiamat.xml:63]: Wis save +4

File: bestiary/rise-of-tiamat.xml [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+				<li><strong>Skills</strong> Deception +4, Stealth +5</li>
```
Comment (ziphmose) [inline bestiary/rise-of-tiamat.xml:64]: Deceptlion +5, Stealth +6

File: bestiary/rise-of-tiamat.xml [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+				<p></p>
+				<li><strong><em>Orb of Dragon's Breath (3/Day). </em></strong><em>Ranged Spell Attack:</em> +7 to hit, range 90 ft., one target. <em>Hit:</em> 27 (6d8) damage of the type to which the dragonsoul has damage resistance.</li>
+			</ul>
+		</description>
+	</element>
+
+	<element name="Dragonwing" type="Bestiary" source="Rise of Tiamat" id="ID_WOTC_ROT_STATBLOCK_DRAGONWING">
+		<description>
+			<h3 style="margin-bottom:-20px;margin-top:-5px">Dragonwing</h3>
+			<p style="margin-bottom:-5px"><em>Medium humanoid (human), neutral evil</em></p>
```
Comment (ziphmose) [inline bestiary/rise-of-tiamat.xml:137]: Lawful Evil in HotDQ

**Review by ziphmose — APPROVED** [review]

**Pull request merged**